src/converter/one/proto/Proto.java

```diff
@@ -6,13 +6,16 @@
 package one.proto;
 
 import java.nio.charset.StandardCharsets;
+import java.nio.ByteBuffer;
 import java.util.Arrays;
 
 /**
  * Simplified implementation of Protobuf writer, capable of encoding
  * varints, doubles, ASCII strings and embedded messages
  */
 public class Proto {
+    private static final int DEFAULT_MAX_LEN_BYTE_COUNT = 3;
```

Comment:
It needs a better name.
5 words with `default` and `max`, `len` and `count`, all within the same name, sounds gibberish.

Reply:
Removed in 2359e43708a736b01714e3842adfd6a91ad6e203